vic-class08-assignment/server.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+const http = require('http');
+const fs = require('fs');
+const stream = require('stream');
+
+http.createServer((req, res) => {
+  if(req.url === '/notes' && req.method === 'GET') {
+    fs.readdirSync(__dirname + '/data', (err, data) => {
+      if (err) {return res.end('err');}
+    });
+    res.writeHead(200, {'Content-Type': 'text/plain'});
+    let file = fs.readdirSync(__dirname + '/data');
```

Comment:
Hey Victor.  Let's try and figure out a way to do this where we aren;t having to read the file twice.  You are doing it here and on line 8.